File: data/src/main/java/com/joxad/zikobot/data/module/spotify_auth/model/SpotifyToken.java
package com.joxad.zikobot.data.module.spotify_auth.model;

import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class SpotifyToken {
    @SerializedName("access_token")
    protected String accessToken;
    @SerializedName("token_type")
    protected String tokenType;
    @SerializedName("expires_in")
    protected int expiresIn;
    @SerializedName("refresh_token")
    protected String refreshToken;
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_auth/model/SpotifyToken.kt
package com.joxad.zikobot.data.module.spotify_auth.model

import com.google.gson.annotations.SerializedName

class SpotifyToken {
    @SerializedName("access_token")
    var accessToken: String? = null
    @SerializedName("token_type")
    var tokenType: String? = null
    @SerializedName("expires_in")
    var expiresIn: Int = 0
    @SerializedName("refresh_token")
    var refreshToken: String? = null
}
